test/e2e/mxGraph.model.test.ts

```diff
@@ -1024,6 +1024,25 @@ describe('mxGraph model', () => {
     expect('inclusive_gateway_id').toBeShape({ kind: ShapeBpmnElementKind.GATEWAY_INCLUSIVE, label: 'Inclusive Gateway', parentId: 'participant_1_id', verticalAlign: 'top' });
     expect('parallel_gateway_id').toBeShape({ kind: ShapeBpmnElementKind.GATEWAY_PARALLEL, label: 'Parallel Gateway', parentId: 'participant_1_id', verticalAlign: 'top' });
     expect('exclusive_gateway_id').toBeShape({ kind: ShapeBpmnElementKind.GATEWAY_EXCLUSIVE, label: 'Exclusive Gateway', parentId: 'participant_1_id', verticalAlign: 'top' });
+    expect('gateway_event_based_id').toBeEventBasedGateway({
+      label: 'Event-Based Gateway',
+      parentId: 'participant_1_id',
+      verticalAlign: 'top',
+      isInstantiating: false,
+    });
+    expect('gateway_event_based_instantiate_id').toBeEventBasedGateway({
+      label: 'Event-Based Gateway Instantiate',
+      parentId: 'participant_1_id',
+      verticalAlign: 'top',
+      isInstantiating: true,
+    });
+    expect('gateway_event_based_instantiate_parallel_id').toBeEventBasedGateway({
```

Comment:
Missing same test with **Exclusive**

Reply:
I will add a test on flownode parsing instead, see my comment about the visual tests for reasons.

Reply:
Ok for the model test